lib/js/components/Skeleton/Skeleton.consts.ts

```diff
@@ -0,0 +1,7 @@
+export const SKELETON_RADIUSES = {
```

Comment:
what would you say about renaming it to `SKELETON_RADIUS_SIZES`?

Reply:
Sure, but let's wait for Ewa in https://bethinkteam.slack.com/archives/CFBLF5RGA/p1721995047212719?thread_ts=1721652437.934089&cid=CFBLF5RGA, in case she wants to change something else about radiuses.